pilot/cmd/pilot-agent/status/server.go

```diff
@@ -59,7 +59,14 @@ var (
 // It's a map from the prober URL path to the Kubernetes Prober config.
 // For example, "/app-health/hello-world/livez" entry contains livenss prober config for
 // container "hello-world".
-type KubeAppProbers map[string]*corev1.HTTPGetAction
+//type KubeAppProbers map[string]*corev1.HTTPGetAction
+
+type Prober struct {
+	HTTPGetAction  *corev1.HTTPGetAction
+	TimeoutSeconds int32
```

Comment:
nit: add ` `json:"timeoutSeconds,omitempty"` and `json:"httpGetAction,omitempty`